app/src/components/MultiSelectInput.tsx

```diff
@@ -137,10 +143,15 @@ const MultiSelectWithCheckbox = ({
                 Option: CustomOption,
               }}
               value={
-                field.value?.map((val: string) => ({
-                  label: t(val),
-                  value: val,
-                })) || []
+                field.value.length === 0
+                  ? preselectedValue.map((val: string) => ({
+                      label: t(val),
+                      value: val,
+                    }))
+                  : field.value?.map((val: string) => ({
+                      label: t(val),
+                      value: val,
+                    }))
```

Comment:
If you can, do this about the returned markup in the head part of the react component to make it more readable.